### PR TITLE
108 update accordion component layout

### PR DESCRIPTION
**Pull request by alininayeh**
### What changed?
 - the accordion component has been updated with the new design
- the accordion component accepts a logo

### CODE REVIEW

**Comment by aniri** [comment]
closes https://github.com/code4romania/taskforce-fe-components/issues/108

File: src/components/accordion/accordion.js [hunk 1]
```diff
@@ -4,28 +4,34 @@ import classNames from "classnames";
...
-        {title}
+        <span className="__accordion-title">{title}</span>
         <div className="__accordion-arrow">
           <ArrowDown />
         </div>
       </div>
-      {showContent && <div className="message-body">{content}</div>}
+      {showContent && (
+        <div className="message-body">
+          {logo && <img className="__accordion-logo" src={logo} />}
```
Comment (moonflare) [inline src/components/accordion/accordion.js]: Hmm, I wouldn't pass the image/logo as a prop, but have it as a child in the `content` prop, or perhaps change the `content` prop to be `children` instead? I think that would more appropriate now that we have more than paragraphs in the content of the accordion.

**Comment by moonflare** [comment]
Hey @alininayeh 👋  It looks good. I left a comment tho about the structure of the props/children of the component

**Comment by moonflare** [comment]
Hey @alininayeh. Are you familiar with rebasing? You have some commits that are from `master` branch and don't belong to the PR, and by rebasing it will leave you only with your commits. Otherwise, please let me know and I can fix it for you.

**Comment by alininayeh** [comment]
> Hey @alininayeh. Are you familiar with rebasing? You have some commits that are from `master` branch and don't belong to the PR, and by rebasing it will leave you only with your commits. Otherwise, please let me know and I can fix it for you.

Hi. I think I will need some help, I don't want to mess anything up :)

**Comment by moonflare** [comment]
> > Hey @alininayeh. Are you familiar with rebasing? You have some commits that are from `master` branch and don't belong to the PR, and by rebasing it will leave you only with your commits. Otherwise, please let me know and I can fix it for you.
> 
> Hi, @moonflare . I think I will need some help, I don't want to mess anything up :)

No worries. I will fix it

**Comment by moonflare** [comment]
> > > Hey @alininayeh. Are you familiar with rebasing? You have some commits that are from `master` branch and don't belong to the PR, and by rebasing it will leave you only with your commits. Otherwise, please let me know and I can fix it for you.
> > 
> > 
> > Hi, @moonflare . I think I will need some help, I don't want to mess anything up :)
> 
> No worries. I will fix it

Fixed. We can see just your commits. I also realized that we want to be backwards compatible with the content prop, we will deprecate it later, but for now I added it back to not break everything that's using it atm.

File: src/components/accordion/accordion.js [hunk 2]
```diff
@@ -4,28 +4,39 @@ import classNames from "classnames";
...
   );
 };
 
 Accordion.propTypes = {
   title: PropTypes.string.isRequired,
-  content: PropTypes.node.isRequired
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node
+  ]).isRequired,
```
Comment (aniri) [inline src/components/accordion/accordion.js]: if the new prop is marked as required, we are not backwards compatible :(

**Pull request merged**